<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>316</width>
    <height>405</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QFormLayout" name="formLayout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::ExpandingFieldsGrow</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="applyFormatLabel">
        <property name="text">
         <string>&lt;p&gt;&lt;a href=&quot;apply&quot;&gt;Apply format&lt;/a&gt;&lt;/p&gt;</string>
        </property>
        <property name="openExternalLinks">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="comboBox">
        <property name="currentIndex">
         <number>3</number>
        </property>
        <item>
         <property name="text">
          <string>All capital letter</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>All lower case letter</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>First letter capital</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>First letter every word capital</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="fileNameLabel">
        <property name="text">
         <string>&lt;p&gt;&lt;b&gt;Name:&lt;/b&gt;&lt;/p&gt;</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLineEdit" name="fileNameLine">
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
          <property name="dragEnabled">
           <bool>true</bool>
          </property>
          <property name="placeholderText">
           <string>file name</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="clearButton1">
          <property name="text">
           <string>...</string>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <widget class="QToolButton" name="toFileName">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>..</string>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QLineEdit" name="toFileNameLine">
          <property name="text">
           <string>%a - %t</string>
          </property>
          <property name="placeholderText">
           <string>to file name</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="clearButton7">
          <property name="text">
           <string>...</string>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="0">
       <widget class="QToolButton" name="fromFileName">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>..</string>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <widget class="QLineEdit" name="fromFileNameLine">
          <property name="text">
           <string>%n-%a-%t</string>
          </property>
          <property name="placeholderText">
           <string>from file name</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="clearButton8">
          <property name="text">
           <string>...</string>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="titleLabel">
        <property name="text">
         <string>&lt;p&gt;&lt;b&gt;Title:&lt;/b&gt;&lt;/p&gt;</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLineEdit" name="titleLine">
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
          <property name="dragEnabled">
           <bool>true</bool>
          </property>
          <property name="placeholderText">
           <string>title</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="clearButton2">
          <property name="text">
           <string>...</string>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="artistLabel">
        <property name="text">
         <string>&lt;p&gt;&lt;b&gt;Artist:&lt;/b&gt;&lt;/p&gt;</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLineEdit" name="artistLine">
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
          <property name="dragEnabled">
           <bool>true</bool>
          </property>
          <property name="placeholderText">
           <string>artist</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="clearButton3">
          <property name="text">
           <string>...</string>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="albumLabel">
        <property name="text">
         <string>&lt;p&gt;&lt;b&gt;Album:&lt;/b&gt;&lt;/p&gt;</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLineEdit" name="albumLine">
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
          <property name="dragEnabled">
           <bool>true</bool>
          </property>
          <property name="placeholderText">
           <string>album</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="clearButton4">
          <property name="text">
           <string>...</string>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="trackLabel">
        <property name="text">
         <string>&lt;p&gt;&lt;b&gt;Track:&lt;/b&gt;&lt;/p&gt;</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QSpinBox" name="trackLine">
        <property name="maximum">
         <number>999</number>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="yearLabel">
        <property name="text">
         <string>&lt;p&gt;&lt;b&gt;Year:&lt;/b&gt;&lt;/p&gt;</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QSpinBox" name="yearLine">
        <property name="maximum">
         <number>3000</number>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QLabel" name="genreLabel">
        <property name="text">
         <string>&lt;p&gt;&lt;b&gt;Genre:&lt;/b&gt;&lt;/p&gt;</string>
        </property>
       </widget>
      </item>
      <item row="9" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLineEdit" name="genreLine">
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
          <property name="dragEnabled">
           <bool>true</bool>
          </property>
          <property name="placeholderText">
           <string>genre</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="clearButton5">
          <property name="text">
           <string>...</string>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="10" column="0">
       <widget class="QLabel" name="commentLabel">
        <property name="text">
         <string>&lt;p&gt;&lt;b&gt;Comment:&lt;/b&gt;&lt;/p&gt;</string>
        </property>
       </widget>
      </item>
      <item row="10" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QPlainTextEdit" name="commentLine">
          <property name="centerOnScroll">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QToolButton" name="clearButton6">
            <property name="text">
             <string>...</string>
            </property>
            <property name="autoRaise">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>316</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>clearButton1</sender>
   <signal>pressed()</signal>
   <receiver>fileNameLine</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>309</x>
     <y>88</y>
    </hint>
    <hint type="destinationlabel">
     <x>281</x>
     <y>88</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearButton2</sender>
   <signal>pressed()</signal>
   <receiver>titleLine</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>309</x>
     <y>178</y>
    </hint>
    <hint type="destinationlabel">
     <x>281</x>
     <y>178</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearButton3</sender>
   <signal>pressed()</signal>
   <receiver>artistLine</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>309</x>
     <y>208</y>
    </hint>
    <hint type="destinationlabel">
     <x>281</x>
     <y>208</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearButton4</sender>
   <signal>pressed()</signal>
   <receiver>albumLine</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>309</x>
     <y>238</y>
    </hint>
    <hint type="destinationlabel">
     <x>281</x>
     <y>238</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearButton5</sender>
   <signal>pressed()</signal>
   <receiver>genreLine</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>309</x>
     <y>324</y>
    </hint>
    <hint type="destinationlabel">
     <x>281</x>
     <y>324</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearButton6</sender>
   <signal>pressed()</signal>
   <receiver>commentLine</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>308</x>
     <y>355</y>
    </hint>
    <hint type="destinationlabel">
     <x>279</x>
     <y>394</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearButton7</sender>
   <signal>pressed()</signal>
   <receiver>toFileNameLine</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>309</x>
     <y>118</y>
    </hint>
    <hint type="destinationlabel">
     <x>281</x>
     <y>118</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearButton8</sender>
   <signal>pressed()</signal>
   <receiver>fromFileNameLine</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>292</x>
     <y>136</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>136</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
